website/src/content/blog/roadmap-2024.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Roadmap 2024
+description: Roadmap 2024, v1.5.0 and new logo
+summary: Roadmap 2024, v1.5.0 and new logo
+authors:
+  - ema
+  - team
+pubDate: 2024-01-06
+coverImage:
+  lightSrc: ../../assets/blog/roadmap-2024/banner-light.png
+  darkSrc: ../../assets/blog/roadmap-2024/banner-dark.png
+  alt: The Prettier challenge banner, with the Biome logo over it
+socialImage: ../../assets/social-logo.png
+---
+## Roadmap 2024
+
+We are thrilled to share what the Core Contributors and Maintainers would like to focus on in 2024.
+
+We want to remind the community that Biome is a project driven by the community, so we can't promise that all the ideas outlined below are going to be shipped.
+
+However, if you're excited about some aspects of the project, and you want to see some of them developed faster than others, you can help us in many ways:
+- [**Be involved in the project and the community**](https://github.com/biomejs). Help us to build those features.
+- [**Sponsor us**](https://opencollective.com/biome). Ask your company to sponsor us. Biome is so fast that can cut down CI times in your company, and save money. Performance is part of our mission. Plus, sponsorship is a good medium of _advertisement_ for your company.
+- [**Improve our documentation with ideas, recipes, or guides**](/guides/getting-started). Translate our documentation, help us to make Biome available to people who aren't proficient in English.
+
+
+### Preface
+
+The project is young and still can't compete against other giants such as Prettier, eslint, webpack, vite, esbuild, etc., although the recent events (sponsors, bounty challenge, Biome being a fork of Rome) showed that the users **have** interest in the project, and we showed those users that we have the tools for fulfilling a need.
+
+Moving small projects from ESLint/Prettier is easy, but moving **big** code bases is challenging and time-consuming. This is a big friction to Biome.
+
+Users have different needs though, so it will be impossible to satisfy all of them, so we want to make sure that they contribute to our project and embrace our philosophy.
```

Comment:
```suggestion
Users have different needs though, so it will be impossible to satisfy all of them. We want to make sure that all features and contributions to our project [embrace our philosophy](/internals/philosophy/) and provide the best experience by default.
```
Took a little creative freedom adding in the last part here, but I wanted to frame it as an opportunity rather than a restriction.